src/elements/tabs/tab-group/tab-group.ts

```diff
@@ -301,7 +309,14 @@ export class SbbTabGroupElement extends LitElement {
           this._selectedTab = tab;
 
           this._tabContentResizeObserver.observe(tab.relatedContent!);
-          this._selectedTabChanged.emit();
+          this._selectedTabChanged.emit({
+            activeIndex: this._getTabs().findIndex((e) => e === tab),
```

Comment:
the call to this._getTabs() could be saved to a const to not call it all the time and should probably be a getter.

Reply:
i changed it to _tabs